Scenes/crew_member.gd

```diff
@@ -12,25 +12,27 @@ var following = false
 func _physics_process(delta: float) -> void:
 	velocity = push_velocity
 	push_velocity = Vector2.ZERO
-	if assignment_target:
-		var distance_to_assignment =  global_position.distance_to(assignment_target.global_position)
-		if distance_to_assignment > 5:
-			var direction = global_position.direction_to(assignment_target.global_position)
-			if (assignment_target is not Player) or (distance_to_assignment > follow_distance):
-				velocity += direction * SPEED
-		else:
-			global_position == assignment_target.global_position
 	
-	if velocity != Vector2.ZERO and push_velocity == Vector2.ZERO:
-		if rad_to_deg(velocity.angle()) < -15 and  rad_to_deg(velocity.angle()) > -165:
-			$AnimatedSprite2D.play("walking_up")
-		else:
-			$AnimatedSprite2D.play("walking_down")
-	else:
-		if following and global_position.direction_to(assignment_target.global_position).y < 0:
-			$AnimatedSprite2D.play("idle_up")
+	if not ($AnimatedSprite2D.animation == "alert" or $AnimatedSprite2D.animation == "acknowledge") or not $AnimatedSprite2D.is_playing():
```

Comment:
Likely need to handle these with states in the future like in https://github.com/Splashy-Inc/splashy-fish-2